src/VisualStudio/CSharp/Impl/LanguageServerClient/CSharpLanguageServerClient.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.VisualStudio.LanguageServer.Client;
+using Microsoft.VisualStudio.Threading;
+using Microsoft.VisualStudio.Utilities;
+using Microsoft.CodeAnalysis.Remote;
+using Microsoft.ServiceHub.Client;
+using Microsoft.CodeAnalysis.Editor;
+
+namespace Microsoft.VisualStudio.LanguageServices.CSharp.LanguageServerClient
+{
+    [ContentType(ContentTypeNames.CSharpContentType)]
+    [Export(typeof(ILanguageClient))]
+    [ExportMetadata("Capability", "WorkspaceStreamingSymbolProvider")]
+    internal class CSharpLanguageServerClient : ILanguageClient
+    {
+        /// <summary>
+        /// Gets the name of the language client (displayed to the user).
+        /// </summary>
+        public string Name => "C# Language Server Client";
```

Comment:
IIRC this is shown to the user on exception. Do we need to localize?

Reply:
yes, this one has to be localized